ietf-scitt-charter.md

```diff
@@ -4,6 +4,8 @@ Over the years, rapid technological advancements have motivated organizations to
 While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size, and complexity of supply chains has led to more frequent and sophisticated supply chain attacks.
 The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate.
 
+Scitt forms a building block that will allow implemeters to compose complex supply chain integrity systems. For example, public computer interface system could report its software composition and that can be compared against known software compositions for such a device, recorded in a public ledger, thus providing an individual using the system with confidence that it will behave as expected, when expected, and nothing/nowhen else.
```

Comment:
The text read to me like the attempt to illustrate how SCITT fits into existing work (namely RATS) and how the bigger picture looks like. I tried to rephrase the text a bit better towards this goal. We need some feedback, if this expositional text renders the charter more or less readable/focused, though :-)

```suggestion
The output of the SCITT WG composes building blocks that allow implementers to compose complex supply chain integrity systems. For example, public computer interface systems are enabled to report their software composition (facilitating, e.g., out from the RATS WG), which in turn can be compared against known software compositions for such devices enabled via SCITT WG output: building blocks that enable the recording in a public append only logs and corresponding proofs. Thus providing an individual using the system with confidence that the system will behave as expected, when expected, and doing nothing else.
```

Reply:
I would change "building blocks" to "a set of standards" to reflect what it is. Building block implies something is already built.  Also, I would drop "complex" as it is a negative word, "supply chain integrity systems" is complex enough.

> SCITT WG aims to deliver a set of standards that allow implementers to compose supply chain integrity systems.

Example could be moved to a separate dedicated section as it adds confusion.

Reply:
I agree with Ivar, we need to correct the statement.
```suggestion
The output of the SCITT WG is a set of standards that define the essential building blocks that enables security of a complex supply chain system. Following these standards will assist implementers to compose complex supply chain system in a secure manner. For example, public computer interface system could report its software composition and that can be compared against known software compositions for such a device, recorded in a public append only transparent registry, thus providing an individual using the system with confidence that it will behave as and when expected, and doing nothing else.
```

Reply:
Pending a few editorial changes, this reads good. Do we want to drop "complex"? Sounds okay to me.

Reply:
```suggestion
The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of a complex supply chain system and assisting implementers in securing deployments. For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in a public append-only transparent registry. Therefore, providing an individual using the system with confidence that it will behave as and when expected and will do nothing else.
```

Reply:
```suggestion
The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of supply chain systems and assisting implementers in securing deployments. For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in a public append-only transparent registry. Therefore, providing an individual using the system with confidence that it will behave as and when expected and will do nothing else.
```

Removed complex as requested. Does that read okay?

Reply:
That reads well.  Fantastic work, everyone.